polkadot-parachain/src/chain_spec/bridge_hubs.rs

```diff
@@ -152,11 +149,9 @@ impl BridgeHubRuntimeType {
 				|_| (),
 			))),
 			BridgeHubRuntimeType::Wococo => Ok(Box::new(wococo::live_config(
```

Comment:
@PierreBesson 
remove also `Ok(Box::new(wococo::live_config(` here

Reply:
Thank you I fixed it.